src/components/Inbox/DirectMessages.js

```diff
@@ -0,0 +1,111 @@
+import React, { Component } from 'react';
+import './directMessages.scss';
+import axios from 'axios';
+
+class DirectMessages extends Component {
+
+
+    markAsRead = () => {
+        axios.post('/api/comment/read', { commentID: this.props.commentID })
+            .then(() => this.props.readFunction(this.props.commentID))
+    }
+
+    markAsUnread = () => {
+        axios.post('/api/comment/unread', { commentID: this.props.commentID })
+            .then(() => this.props.readFunction(this.props.commentID))
+    }
+
+    toggleReadClick = () => {
+        console.log("toggleReadClick=>", this.props.commentRead)
+        if (this.props.commentRead) { this.markAsUnread() }
+        else { this.markAsRead() }
+
+
+    }
+
+    handleClickLikes = () => {
+        axios.post('/api/comment/like')
```

Comment:
It seems like this method trying to add a like when the like icon clicked. This could be easily abused by smashing the like button and freak out the server will hundred of calls.